src/api.jl

```diff
@@ -4795,7 +4804,7 @@ XPRSaddrows,
 XPRSdelcols,
 XPRSchgcoltype.
 """
-function addcols(prob::XpressProblem, _dobj::Vector{Float64}, _mstart::Vector{Int}, _mrwind::Vector{Int}, _dmatval::Vector{Float64}, _dbdl::Vector{Float64}, _dbdu::Vector{Float64})
+function addcols(prob::XpressProblem, _dobj::Vector{Float64}, _mstart::Vector{Cint}, _mrwind::Vector{Cint}, _dmatval::Vector{Float64}, _dbdl::Vector{Float64}, _dbdu::Vector{Float64})
```

Comment:
shoudl be Vector{I} where I<:Integer right? take  alook a the last version of the PR

Reply:
`Vector{I} where I <: Integer` is the same as `Vector{<:Integer}` the I is only required if we use `I` for something in the function.

Reply:
ok!
lets do:

_mstart::Vector{<:Integer}, _mrwind::Vector{<:Integer},